webpack.mix.js

```diff
@@ -0,0 +1,13 @@
+const mix_ = require('laravel-mix');
+
+mix_.setPublicPath('./dist')
+  .sass('./assets/scss/login.scss', 'css/login.css')
+  .options({
+    processCssUrls: false
+  });
+
```

Comment:
I know the file was pretty small to start with so maybe not worth it but isn't there a function like .minify('dist/css/login.css') you can use to minify the css?